clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -74,9 +74,18 @@ static droption_t<std::string>
               "Specifies the file path of the PT raw trace. Please run the "
               "libipt/script/perf-read-aux.bash script to get PT raw trace file from the "
               "data generated by the perf record command.");
+static droption_t<std::string>
+    op_elf(DROPTION_SCOPE_FRONTEND, "elf", "", "[Optional] Path of the elf file",
```

Comment:
s/Path of/Path to/
same below

s/You must specify/This must be specified/
s/converting short traces/converting traces/
add e.g. kernel-only traces and short user traces.

Reply:
Done.